### PR TITLE
feat(tiptap): add text alignment

### LINKED ISSUE #1024: feat(tiptap): add text align
Users wish to be able to align invitations texts / event descriptions. See https://tiptap.dev/api/extensions/text-align.

### PR DESCRIPTION
**Pull request by myyxl**
The style issue still needs to be resolved as discussed in #1024

Closes #1024

### CODE REVIEW

File: src/components/_/TipTap.vue [hunk 1]
```diff
@@ -236,6 +269,9 @@ de:
   redo: Wiederholen
   strike: Durchgestrichen
   undo: Rückgängig machen
+  formatAlignLeft: Linksbündig
+  formatAlignRight: Rechtsbündig
+  formatAlignCenter: Mittig
```
Comment (dargmuesli) [inline src/components/_/TipTap.vue:274]: Would you keep the nice alphabetical sorting of translation keys? :pleading_face:

File: src/package.json [hunk 2]
```diff
@@ -143,7 +143,8 @@
     "vue-gtag": "2.0.1",
     "vue-qrcode-reader": "5.3.4",
     "vue-tsc": "1.8.10",
-    "workbox-precaching": "7.0.0"
+    "workbox-precaching": "7.0.0",
+    "@tiptap/extension-text-align": "^2.1.8"
```
Comment (dargmuesli) [inline src/package.json:147]: `pnpm add -DE ...`: `-D` installs it as `devDependency` `-E` as exact version (without the `^`, is taken care of by renovate).
As this package does not need to provide its dependencies to other packages, `devDependency`s are fine.

File: src/components/_/TipTap.vue (hunk 1 above)
Comment (dargmuesli) [reply src/components/_/TipTap.vue]: done

File: src/components/_/TipTap.vue [hunk 3]
```diff
@@ -134,6 +134,33 @@
...
+      <div class="flex gap-1">
+        <ButtonIcon
+          :aria-label="t('formatAlignCenter')"
+          :title="t('formatAlignCenter')"
+          @click="editor?.chain().focus().setTextAlign('center').run()"
+        >
+          <IconCenterAlignment />
+        </ButtonIcon>
+      </div>
+      <div class="flex gap-1">
```
Comment (dargmuesli) [inline src/components/_/TipTap.vue]: The buttons belong to the same group, so they do not all need to be wrapped with `gap-1`.

**Comment by dargmuesli** [comment]
Are you fine with my changes? @myyxl

File: src/components/_/TipTap.vue (hunk 3 above)
Comment (dargmuesli) [reply src/components/_/TipTap.vue]: done

**Comment by myyxl** [comment]
Looks good :+1:

**Review by dargmuesli — APPROVED** [review]

**Pull request merged**